views.go

```diff
@@ -144,9 +144,10 @@ func (lk *logkeeper) createBuild(w http.ResponseWriter, r *http.Request) {
 	}
 
 	payload := struct {
-		Builder  string `json:"builder"`
-		BuildNum int    `json:"buildnum"`
-		TaskID   string `json:"task_id"`
+		Builder       string `json:"builder"`
+		BuildNum      int    `json:"buildnum"`
+		TaskID        string `json:"task_id"`
+		TaskExecution int    `json:"task_execution"`
```

Comment:
Unfortunately this has to be `execution` since my change in resmoke passes in just `execution` instead of `task_exceution`. Sorry about that, I wasn't thinking :(.

Reply:
@julianedwards Done!